tracetools_trace/tracetools_trace/trace.py

```diff
@@ -125,15 +134,20 @@ def init(
         display_list=display_list,
     )
 
+    if is_runtime_session:
+        session_name = session_name + '-runtime'
```

Comment:
I see a lot of `session_name + '-runtime'` and `session_name.removesuffix('-runtime')`, in this file and in other files. Ideally, we'd want this logic (even though it's pretty trivial) in a single place for maintainability. However, it's hard to know how it could be improved without seeing the full implementation. Therefore, I will wait until then to take a closer look and potentially recommend a solution